dspy/streaming/streaming_listener.py

```diff
@@ -63,14 +67,19 @@ def receive(self, chunk: ModelResponseStream):
             end_identifier = self.json_adapter_end_identifier
 
             start_indicator = '"'
+        elif isinstance(settings.adapter, XMLAdapter):
+            start_identifier = self.xml_adapter_start_identifier
+            end_identifier = self.xml_adapter_end_identifier
+
+            start_indicator = "<"
```

Comment:
Can we also make start_indicator an instance variable like we do for start_identifier and end_identifier?